WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StoreStatsChart.swift

```diff
@@ -0,0 +1,179 @@
+import Charts
+import SwiftUI
+
+/// Chart for store performance built with Swift Charts.
+/// Technical notes: paNNhX-tA-p2
```

Comment:
Suggestion: For future reference, consider adding a link to this PR as a comment to the P2.

Reply:
Good idea! I added a comment here: paNNhX-tA-p2#comment-807